gbpservice/nfp/configurator/agents/firewall.py

```diff
@@ -28,7 +28,7 @@
 rest_timeout = [
     cfg.IntOpt(
         'rest_timeout',
-        default=240,
+        default=60,
```

Comment:
If this needs to be common, we'll have to put it in a common location rather than FW agent. Else, just like LB, VPN shall have to use it's own config option and this FW parameter has to be renamed.